conans/client/cmd/download.py

```diff
@@ -13,12 +13,16 @@ def download(app, ref, package_ids, recipe):
     hook_manager = app.hook_manager
     assert(isinstance(ref, RecipeReference))
     scoped_output = ScopedOutput(str(ref), ConanOutput())
-    remote = app.selected_remote
-    if not remote:
+    # TODO: for download we should only accept one remote ?
```

Comment:
I would say yes, the command is a bit weird, downloading from N is even more weird.